packages/bootstrap/scss/scheduler/_variables.scss

```
@@ -1,92 +1,235 @@
+/// @group scheduler
 $kendo-scheduler-event-hover-text: null !default;
+/// The border color of the hovered Scheduler event.
+/// @group scheduler
 $kendo-scheduler-event-hover-border: null !default;
+/// The gradient of the hovered Scheduler event.
+/// @group scheduler
 $kendo-scheduler-event-hover-gradient: null !default;
+/// The shadow of the hovered Scheduler event.
+/// @group scheduler
 $kendo-scheduler-event-hover-shadow: null !default;
 
+/// The background color of the selected Scheduler event.
+/// @group scheduler
 $kendo-scheduler-event-selected-bg: $kendo-selected-bg !default;
+/// The text color of the selected Scheduler event.
+/// @group scheduler
 $kendo-scheduler-event-selected-text: $kendo-selected-text !default;
+/// The border color of the selected Scheduler event.
+/// @group scheduler
 $kendo-scheduler-event-selected-border: null !default;
+/// The gradient of the selected Scheduler event.
+/// @group scheduler
 $kendo-scheduler-event-selected-gradient: null !default;
+/// The shadow of the selected Scheduler event.
+/// @group scheduler
 $kendo-scheduler-event-selected-shadow: inset 0 0 0 3px rgba( white, .5 ) !default;
 
+/// The shadow of the ongoing Scheduler event.
+/// @group scheduler
 $kendo-scheduler-event-ongoing-shadow: inset 0px 0px 0px 1px #ff0000 !default;
 
+/// The horizontal padding of the Scheduler cell.
+/// @group scheduler
 $kendo-scheduler-cell-padding-x: k-spacing(3) !default;
+/// The vertical padding of the Scheduler cell.
+/// @group scheduler
 $kendo-scheduler-cell-padding-y: k-spacing(3) !default;
+/// The height of the Scheduler cell.
+/// @group scheduler
 $kendo-scheduler-cell-height: $kendo-line-height-em !default;
+/// The width of the Scheduler date column.
+/// @group scheduler
 $kendo-scheduler-datecolumn-width: 12em !default;
+/// The width of the Scheduler time column.
+/// @group scheduler
 $kendo-scheduler-timecolumn-width: 11em !default;
 
+/// The background color of the non-working hours in the Scheduler.
+/// @group scheduler
 $kendo-scheduler-nonwork-bg: if($kendo-enable-color-system, k-color( base-subtle ), k-try-shade( $kendo-scheduler-bg, .5 )) !default;
+/// The text color of the non-working hours in the Scheduler.
+/// @group scheduler
 $kendo-scheduler-nonwork-text: null !default;
 
+/// The background color of the weekends in the Scheduler.
+/// @group scheduler
 $kendo-scheduler-weekend-bg: null !default;
+/// The text color of the weekends in the Scheduler.
+/// @group scheduler
 $kendo-scheduler-weekend-text: null !default;
 
+/// The background color of the other months in the Scheduler.
+/// @group scheduler
 $kendo-scheduler-othermonth-bg: if($kendo-enable-color-system, k-color( base-subtle ), k-try-shade( $kendo-scheduler-bg, .5 )) !default;
+/// The text color of the other months in the Scheduler.
+/// @group scheduler
 $kendo-scheduler-othermonth-text: null !default;
 
+/// The horizontal padding of the Scheduler year view.
```

Make this change for all occurrences.
```suggestion
/// The horizontal padding of the year view in the Scheduler.
```